README.md

```diff
@@ -5,15 +5,110 @@
 Sumo Logic Distro of [OpenTelemetry Collector][otc_link] built with
 [opentelemetry-collector-builder][otc_builder_link].
 
+[otc_link]: https://github.com/open-telemetry/opentelemetry-collector
+[otc_builder_link]: https://github.com/open-telemetry/opentelemetry-collector-builder
+
 **This software is currently in beta and is not recommended for production environments.**
 
 ## Usage
 
 See the [documentation](docs/README.md).
 
+## Built-in Components
+
+This sections represents the supported components of Sumo Logic OT distro.
+The highlighted components below are developed by Sumo Logic in-house.
```

Comment:
I don't think that's going to fly, IMO that should be just in a separate tables: `Sumo` and `upstream`.